src/pages/Pingers/Pingers.tsx

```diff
@@ -0,0 +1,207 @@
+import { useQuery, useMutation, gql } from "@apollo/client";
+import { loader } from "graphql.macro";
+import React, { useCallback } from "react";
+import RegionSelector from "../../components/RegionSelector";
+import { useParams } from "react-router-dom";
+import {
+  Button,
+  ButtonGroup,
+  Icon,
+  Label,
+  LabelGroup,
+  List,
+  ListItem,
+  Modal,
+  ModalContent,
+  ModalHeader,
+  Segment,
+} from "semantic-ui-react";
+import Form, { PingerSchema } from "components/Form";
+import { TRANSLATION_MAP } from "../../shared/l10n";
+
+const GET_PINGERS = loader("../../graphql/get-pingers.graphql");
+const CREATE_PINGER = loader("../../graphql/create-pinger.graphql");
+const UNSUBSCRIBE_PINGER = loader("../../graphql/unsubscribe-pinger.graphql");
+
+export default function Pingers() {
+  const { id, unsubscribe_key } =
+    useParams<{ id: string; unsubscribe_key: string }>();
+
+  const [selectedPinger, setSelectedPinger] =
+    React.useState<PingerSchema | null>(null);
+
+  const unselectPinger = React.useCallback(() => setSelectedPinger(null), []);
+
+  const pingers = useQuery<{
+    pingers: { results: ReadonlyArray<PingerSchema> };
+  }>(GET_PINGERS, {
+    variables: { id, unsubscribe_key },
+    notifyOnNetworkStatusChange: true,
+    errorPolicy: "all",
+  });
+
+  if (pingers.loading) {
+    return <h1>Loading...</h1>;
+  }
+
+  if (pingers.error) {
+    return <h1>Error: {JSON.stringify(pingers.error)}</h1>;
+  }
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Would you mind just `console.log`ing the error (and maybe sending to bugsnag too) instead of showing it in the UI. The users shouldn't see the error messages. So ideally just a friendly error message is enough.

Reply:
I'm not fluent in graphql and bugsnag, but it seems to me that this should already handle reporting to it:
https://github.com/dmitrijs-balcers/pinger-app/blob/1acc627fc140d14af5403eebe23f942c51bfa8fb/src/shared/apollo-client.ts#L10